packages/react-components/src/organisms/MainNavigation.tsx

```diff
@@ -43,13 +43,16 @@ const listStyles = css({
 });
 
 export interface MainNavigationProps {
-  networkHref: string;
+  discoverASAPHref: string;
```

Comment:
Should we name this
```suggestion
  discoverAsapHref: string;
```
? That's how I (and I thought most people?) have been treating acronyms in camel case, so that it's easy to identify as one word inside the identifier. @kmaid how have you been doing this?

Reply:
Same